src/Azure.Functions.Cli/Helpers/SecurityHelpers.cs

```diff
@@ -121,10 +121,9 @@ internal static async Task<X509Certificate2> CreateCertificateOpenSSL()
             return new X509Certificate2($"{certFileNames}certificate.pfx", DEFAULT_PASSWORD);
         }
 
-        // Suppressing CodeQL warning because this code is only used to validate blob content. Not used to encrypt any secrets
+
+        // CodeQL [SM02196] intent is to validate blob content. Not used to encrypt any secrets
```

Comment:
This is on the wrong line. It needs to be on same line as `using (var md5 = MD5.Create())` or be immediately above it.

``` CSharp
// CodeQL [SM02196] intent is to validate blob content. Not used to encrypt any secrets
using (var md5 = MD5.Create())
```

OR

``` CSharp
using (var md5 = MD5.Create()) // CodeQL [SM02196] intent is to validate blob content. Not used to encrypt any secrets
```